test/utils/test_unbatch_edge_index.py

```diff
@@ -0,0 +1,17 @@
+import torch
```

Comment:
Can we move this test to `test_unbatch.py`?

Reply:
Sure